components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationMetadataService.java

```diff
@@ -151,6 +156,58 @@ public OIDCMetaData getOIDCMetadata() {
         OIDCMetaData oidcMetaData = new OIDCMetaData();
         OAuthAdminServiceImpl oAuthAdminService = ApplicationManagementServiceHolder.getOAuthAdminService();
 
+        List<String> tokenEpAuthMethods = Arrays.asList(OAuth2Util.getSupportedClientAuthMethods());
+        List<ClientAuthenticationMethod> supportedClientAuthenticationMethods = new ArrayList<>();
+        for (String tokenEpAuthMethod : tokenEpAuthMethods) {
+            ClientAuthenticationMethod clientAuthenticationMethod = new ClientAuthenticationMethod();
+            clientAuthenticationMethod.setName(tokenEpAuthMethod);
+            if (tokenEpAuthMethod.equals("client_secret_basic")) {
+                clientAuthenticationMethod.setDisplayName("Client Secret Basic");
+            } else if (tokenEpAuthMethod.equals("client_secret_post")) {
+                clientAuthenticationMethod.setDisplayName("Client Secret Post");
+            } else if (tokenEpAuthMethod.equals("private_key_jwt")) {
+                clientAuthenticationMethod.setDisplayName("Private Key JWT");
+            } else if (tokenEpAuthMethod.equals("tls_client_auth")) {
+                clientAuthenticationMethod.setDisplayName("Mutual TLS");
+            } else {
+                clientAuthenticationMethod.setDisplayName(tokenEpAuthMethod);
+            }
+            supportedClientAuthenticationMethods.add(clientAuthenticationMethod);
+        }
+        oidcMetaData.setTokenEndpointAuthMethod(
+                new ClientAuthenticationMethodMetadata().options(supportedClientAuthenticationMethods));
+        List<String> tokenEpSigningAlgorithms = IdentityUtil
+                .getPropertyAsList(ApplicationManagementConstants.TOKEN_EP_SIGNATURE_ALGORITHMS_SUPPORTED);
+        oidcMetaData.setTokenEndpointSignatureAlgorithm(new MetadataProperty()
+                .options(tokenEpSigningAlgorithms));
+        List<String> idTokenSigningAlgorithms = new ArrayList<>();
+        idTokenSigningAlgorithms.add("None");
```

Comment:
Use a constant for "None" and reuse

Reply:
Fixed in 5285513bf8257aa1ae3c79093209271a18f36da1